src/asciidoc/docs/develop/testingTemplates.adoc

```diff
@@ -0,0 +1,171 @@
+= Testing templates
+:page-slug: testing-templates
+:page-description: Guide to testing Koji templates.
+
+There are many powerful testing tools built into the Koji platform.
+Typically, you will begin testing in the Koji editor.
+Later, you'll want to publish the template as *Unlisted*, so you can test it in a production environment before giving access to general users.
+Some of the most common testing methods are listed below.
+
+== Testing in the Koji editor
+
+The Koji editor provides various ways to preview a template's behavior.
+
+=== Previewing the Koji
+
+The *Live preview* tab allows you to toggle between *Preview* mode, which shows what the Koji looks like when running, and *Editing* mode, which allows you to perform remixing in the preview.
+
+The *Remote* tab offers several additional ways of previewing your Koji.
+
+* You can test it on your mobile device by scanning a QR code.
+Logs from the mobile device appear in a pane in the editor.
+
+* You can open the Koji in a new tab.
+Logs from the tab appear in a pane in the editor.
+
+* You can open it in  *Live player*.
+This opens another browser window where the Koji is displayed in an iframe, just as the published Koji will display.
+You can also use *Live player* to test in-app purchases.
+(See <<_testing_in_app_purchases>>.)
```

Comment:
As mentioned in https://github.com/madewithkoji/docs/pull/109#discussion_r536308025, remove the parenthetical link. If you need to call out a page title, use a regular sentence without extra punctuation. Please fix this formatting throughout the page. In this case, you might consider a contextual link to avoid redundancy with the text.